src/Statement.php

```diff
@@ -266,10 +266,14 @@ public function getAuthority()
      * occurred.
      *
      * @return \DateTime The timestamp
+     *
+     * @deprecated since 1.1.1, to be removed in 3.0
```

Comment:
please change this to `1.2`, I don't want to introduce new deprecations in a patch release